ombpdf/footnotes.py

```diff
@@ -21,6 +22,11 @@
 def is_significantly_smaller(size, normal_size, gap=0.2):
     return float(normal_size - size) > float(gap)
 
+def line_contains_big_chars(line, doc):
+    for char in line:
+        if char.fontsize.size + Decimal('0.5') >= doc.paragraph_fontsize.size:
```

Comment:
Introduced this "fudge factor" of 0.5 to account for lines that might technically be bigger than the paragraph size, but are still within 0.5 points of it.